src/main/java/Item.java

```diff
@@ -53,11 +75,37 @@ public void setItemDescription(String description) {
     public void setDesiredPrice(double newDesiredPrice) {
         this.desiredPrice = newDesiredPrice;
     }
+
+    public void setReviewStars(double reviewStars) { this.reviewStars = reviewStars; }
+    public void setReviewCount(int reviewCount) {
+        this.reviewCount = reviewCount;
+    }
+
     public void displayItemInConsole(int ranking){
         System.out.println("------------------------------------------");
         System.out.println("(" + ranking + "): " + itemName);
         System.out.println("Price: " + itemPrice);
         System.out.println("Date added: " + dateAdded);
+        System.out.println("Last Updated: " + dateLastUpdated);
+        System.out.println("Description:" + itemDescription);
         System.out.println("------------------------------------------");
     }
+
+    public void updatePrice() throws IOException{
+        try {
+            Document doc = Jsoup.connect(url).timeout(10000).userAgent("Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/107.0.0.0 Safari/537.36").get();
```

Comment:
Could you explain what this does please? By possibly adding a comment for future reference as I'm not sure what the long string of text does.

Reply:
that line opens a connection to the item's amazon webpage using a specific user agent that bypasses captcha